config/mappings/test_mapping_config.py

```diff
@@ -0,0 +1,13 @@
+import json
+from jsonschema import validate
+from mapping_schema import top_level_assessment_mapping_schema as schema
+
+with open("config/mappings/mapping.json", "r") as mapping_file:
+    mapping = json.load(mapping_file)
+
+try:
+    validate(instance=mapping, schema=schema)
+except Exception:
+    raise Exception
+finally:
+    print("Mapping conforms with schema.")
```

Comment:
Validate does this for us!

In the future there will be many mapping files. So you could loop through these files using:
```python
for file in os.listdir(directory):
   mapping = json.load(mapping_file)
   validate(instance=mapping, schema=schema)
# Runs if nothing weird happens in for loop.
else:
   print("All mappings are valid!")
```

```suggestion
validate(instance=mapping, schema=schema)
print("Mapping conforms with schema.")
```